axelor-human-resource/src/main/java/com/axelor/apps/hr/rest/dto/ExpenseLinePostRequest.java

```diff
@@ -291,4 +301,11 @@ public ProjectTask fetchProjectTask() {
     }
     return ObjectFinder.find(ProjectTask.class, projectTaskId, ObjectFinder.NO_VERSION);
   }
+
+  public List<Long> fetchEmployeeList() {
+    if (employeeIdList == null || employeeIdList.isEmpty()) {
+      return null;
+    }
+    return employeeIdList;
+  }
```

Comment:
Make it return a List of Employee, and do it also for the PUT DTO.
You can see an example in ExpensePostRequest#fetchExpenseLines.